builtin/settingtypes.txt

```diff
@@ -1842,15 +1842,15 @@ mgvalleys_np_inter_valley_slope (Valley slope) noise_params_2d 0.5, 0.5, (128, 1
 
 [*Advanced]
 
-#    Size of chunks to be generated at once by mapgen, stated in mapblocks (16 nodes).
+#    Size of chunks to be generated at once by Mapgen, stated in MapBlocks (16 nodes).
 #    Increasing this by 1 almost doubles the 3D noise calculation load.
 #    High values can cause noise calculation to overload.
-#    Values smaller than 5 cause a terrain bug in mgv6.
+#    Values smaller than 5 cause a terrain bug in MGv6.
```

Comment:
I think this should remain as is, since `mgv6` is the actual name of Mapgen v6.

Reply:
My mistake, best would be 'Mapgen V6', capitalisation because it is a name.